index.css

```diff
@@ -20,30 +46,40 @@ p{
     font-size: 30px;
     margin-left: 20px;
 }
+/*End Brand block  */
 
-.page .header{
-    width: 100%;
-    margin-bottom: 5px;
-    border: 1px solid red;
-    padding-left: 60px;
+/* Start Navigation block*/
+.navigation{
+    width: 33%;
+    display: inline-block;
+    float: right;
+}
+.navigation .navbar-nav{
+    display: flex;
+    height: auto;
 }
 
-.page .content{
-    width: 100%;
-    height: 500px;
-    margin-bottom: 5px;
-    border: 1px solid red;
+.navigation .navbar-nav .navbar-nav-item{
+    list-style: none;
+    height: auto;
+    width: 60px;
+    padding: 30px 15px 20px 15px;
 }
 
-.page .footer{
-    width: 100%;
-    height: 100px;
-    text-align: center;
-    border: 1px solid red;
+.navigation .navbar-nav .navbar-nav-text{
```

Comment:
I think, navbar-nav  does not make scene in this case. It should be main-menu or something like that.
main-menu is a type of nav, and not all navs are main-menu, main menu is more specified